### PR TITLE
Create sj-roh's first post with a trinket and markdown formatting

### PR DESCRIPTION
**Pull request by sj-roh**

### CODE REVIEW

**Comment by apurva-shah515** [comment]
Post looks great, love the breakdown!

File: _posts/sj-roh/2021-10-20-sj-roh-trinket-post.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+'@' symbol, followed by one or more non-whitespace characters, ending with a letter from each line.
+
+```python
+for line in hand:
+    line = line.rstrip()
+    x = re.findall('@\S+[a-z]',line)
+```
+Lastly, I run another for loop to go through the results from the regular expression, and then append each line of domain
+to the 'domains' open list. I must call the first element within the returned values for x because each line is already 
+in a list, and the findall() function automatically returns values in a list as well. Thus, I call the first elemeent to
```
Comment (apurva-shah515) [inline _posts/sj-roh/2021-10-20-sj-roh-trinket-post.md]: Could use the backtick function to show the findall() method as code possibly

Comment (MishaCienfuegos) [reply _posts/sj-roh/2021-10-20-sj-roh-trinket-post.md]: Looks like SJ accepted this good suggestion and modified this file. Resolving this conversation.

**Review by MishaCienfuegos — APPROVED** [review]
Nice use of markdown within your text to format small bits of code like variable names. It really makes things clearer!

**Pull request merged**